MobileSyncExplorerSwift/MobileSyncExplorerSwift/ViewControllers/RootViewController.swift

```diff
@@ -1,315 +0,0 @@
-/*
- ViewController.swift
- MobileSyncExplorerSwift
-
- Created by Nicholas McDonald on 1/16/18.
-
- Copyright (c) 2018-present, salesforce.com, inc. All rights reserved.
- 
- Redistribution and use of this software in source and binary forms, with or without modification,
- are permitted provided that the following conditions are met:
- * Redistributions of source code must retain the above copyright notice, this list of conditions
- and the following disclaimer.
- * Redistributions in binary form must reproduce the above copyright notice, this list of
- conditions and the following disclaimer in the documentation and/or other materials provided
- with the distribution.
- * Neither the name of salesforce.com, inc. nor the names of its contributors may be used to
- endorse or promote products derived from this software without specific prior written
- permission of salesforce.com, inc.
- 
- THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR
- IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
- FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
- CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
- DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
- DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
- WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
- WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
- */
-
-import UIKit
-import CoreGraphics
-import MobileSync
-
-class RootViewController: UniversalViewController {
-    
-    weak var presentedActions:AdditionalActionsViewController?
-    weak var logoutAlert:UIAlertController?
-    
-    fileprivate var searchText:String = ""
-    fileprivate let tableView = UITableView(frame: .zero, style: .plain)
-    fileprivate let searchController = UISearchController(searchResultsController: nil)
-    fileprivate var sObjectsDataManager = SObjectDataManager(dataSpec: ContactSObjectData.dataSpec()!)
-    
-    deinit {
-        NotificationCenter.default.removeObserver(self)
-    }
-    
-    init() {
-        super.init(nibName: nil, bundle: nil)
-    }
-    
-    required init?(coder aDecoder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
-    }
-    
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        self.view.backgroundColor = UIColor.salesforceSystemBackground
-        
-        NotificationCenter.default.addObserver(self,
```

Comment:
I tested the new code with an alert up and passcode went over it so I didn’t bring this over, let me know if there’s anything else I should verify